src/zc/form/browser/ftests.py

```diff
@@ -94,6 +93,8 @@ def test_suite():
                'getRootFolder': ZCFormLayer.getRootFolder},
         optionflags=(doctest.NORMALIZE_WHITESPACE
                      | doctest.ELLIPSIS
+                     | doctest.REPORT_ONLY_FIRST_FAILURE
```

Comment:
Maybe better omit this line. If someone wants to use this feature he can use `-1` on the command line. It is rather surprising if after fixing an error in a doctest another one occurs wich was hidden by this option.

There is another occurrence of that option in this PR.

Reply:
Unfortunately `-1` on the command line tends not to work, because doctest is a bit stupid (it sees one report formatting option in `optionflags`, such as REPORT_NDIFF, and decides to ignore all global report options, including `REPORT_ONLY_FIRST_FAILURE`.

(Multiple doctest failures in one report have been rarely useful for me and often just plain annyoing.)

Reply:
Ah, that's why it never worked! I actually introduced this line, because there were to many errors all the time an `-1` did not work. Next time, I will sum this up under `doctest.COMPARISON_FLAGS` :wink: